source/common/stats/stats_impl.cc

```diff
@@ -23,29 +59,28 @@ void TimerImpl::TimespanImpl::complete(const std::string& dynamic_name) {
 }
 
 RawStatData* HeapRawStatDataAllocator::alloc(const std::string& name) {
-  RawStatData* data = new RawStatData();
-  memset(data, 0, sizeof(RawStatData));
+  RawStatData* data = static_cast<RawStatData*>(::calloc(RawStatData::size(), 1));
```

Comment:
nit: any reason to not just use malloc?

Reply:
It needs to be zero-initialized.  I've always used calloc for that, even when I don't need it to multiply by an element-count.

Reply:
Ah OK. I would just add comment about zero initialize.